confmap/README.md

```diff
@@ -93,6 +93,91 @@ The `Resolve` method proceeds in the following steps:
 4. For each "Converter", call "Convert" for the "result".
 5. Return the "result", aka effective, configuration.
 
+#### (Experimental) Append merging strategy for lists
+
+You can opt-in to experimentally combine slices instead of discarding the existing ones by enabling `confmap.enableMergeAppendOption` feature flag. Lists are appended in the order in which they appear in their configuration sources.
```

Comment:
```suggestion
You can opt-in to experimentally combine slices instead of discarding the existing ones by enabling the `confmap.enableMergeAppendOption` feature flag. Lists are appended in the order in which they appear in their configuration sources.
```